Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -858,3 +858,39 @@ def generate_login_url(client: MicrosoftClient) -> CommandResults:
 and paste it in your instance configuration under the **Authorization code** parameter.
     """
     return CommandResults(readable_output=result_msg)
+
+
+def get_from_args_or_params(args: Dict[str, Any], params: Dict[str, Any], key: str) -> Any:
+    """
+    Get a value from args or params.
+    this function is used in commands that have a value that can be provided in the instance parameters or in the command,
+    e.g in azure-key-vault-delete 'subscription_id' can be provided in the instance parameters or in the command.
+    Args:
+        args (Dict[str, Any]): Demisto args.
+        params (Dict[str, Any]): Demisto
+        key (str): Key to get.
+    """
+    if value := args.get(key, params.get(key)):
+        return value
+    else:
+        raise Exception(f'No {key} was provided. Please provide a {key} in the instance parameters or in the command')
+
+
+def arg_to_tag(arg):
+    """
+    Convert a tag argument as string to a tag dictionary
```

Comment:
According to the code is a dict converting to a string and not the opposite.

Reply:
Fixed.